src/Serialization/AuxiliaryData.js

```diff
@@ -15,6 +15,9 @@ exports.setTxAuxiliaryData = setter('auxiliary_data');
 exports.newAuxiliaryData = () =>
     lib.AuxiliaryData.new();
 
+exports.auxiliaryDataHashBytes = auxiliaryData =>
```

Comment:
I think this should be renamed to `_hashAuxiliaryData` or something similar. The first thing I think of is converting an `AuxiliaryDataHash` to bytes, not hashing the `AuxiliaryData`